packages/attack-http-requests/src/index.test.ts

```diff
@@ -0,0 +1,51 @@
+import HttpAttack from './index';
+import got from 'got';
+
+const URL = 'http://postman-echo.com';
+
+const makeRequest = async () => {
+  try {
+    await got.get(`${URL}/get?foo1=bar1&foo2=bar2`);
+    return true;
+  } catch (e) {
+    return false;
+  }
+};
+
+const createXRequests = (num: number) =>
+  Promise.all(new Array(num).fill(makeRequest()));
+
+describe('when configuration disabled', () => {
+  it('requests always return true', async () => {
+    const results = await createXRequests(10);
+    expect(results).not.toContain(false);
+  });
+});
+
+describe('when attack is stopped', () => {
+  it('no requests fail', async () => {
+    const attack = new HttpAttack();
+    attack.start();
+    attack.stop();
+    const results = await createXRequests(10);
+    expect(results).not.toContain(false);
+  });
+});
+
+describe('if ignoreUrlPatterns config is passed', () => {
+  test('the requests do not fail', async () => {
```

Comment:
```suggestion
  it('the requests do not fail', async () => {
```

and getting super-nitpicky and don't really care, but the conditions are best written as if you're saying "it does not fail the requests" instead of "it the requests do not fail"
```suggestion
  it('does not fail the requests', async () => {
```